ie_serving/schemas.py

```diff
@@ -33,3 +33,34 @@
     'required': ['all'],
     'properties': {'all': {'type': 'object'}},
 }
+
+models_config_schema = {
+    'definitions': {
+        'model_config': {
+            'type': 'object',
+            'required': ['config'],
+            'properties': {
+                'config': {
+                    'type': 'object',
+                    'required': ['name', 'base_path'],
+                    'properties': {
+                        'name': {'type': 'string'},
+                        'base_path': {'type': 'string'},
+                        'batch_size': {'type': ['integer', 'string']},
```

Comment:
Batch size should be only `int` type

Reply:
Please also update type of `batch_size` parameter for one model path

Reply:
`batch_size` value can also be  "auto" for automatic batch size changes